File: B/Python/Disk.py
# Kelas Disk
class Disk:
    __type = ""
    __capacity = ""
    __price = 0

    # Constructor
    def __init__(self, type="", capacity ="", price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    # Metode Setter
    def setType(self, type):
        self.__type = type

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def setPrice(self, price):
        self.__price = price

    # Metode Getter
    def getType(self):
        return self.__type
    
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price
    
    # Metode Print
    def printDisk(self):
        print("===============================")
        print("Disk:")
        print("===============================")
        print("Tipe         : " + self.getType())
        print("Kapasitas    : " + self.getCapacity())
        print("Harga        : " + "Rp" + str(self.getPrice()))

File: B/Java/Disk.java
// Kelas Disk
class Disk
{
    // Atribut Private
    private String type;
    private String capacity;
    private int price;

    // Constructor
    Disk()
    {
        
    }

    // Constructor Input
    public Disk(String type, String capacity, int price)
    {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    // Metode Getter dan Setter
    public String getType()
    {
        return type;
    }

    public void setType(String type)
    {
        this.type = type;
    }

    public String getCapacity()
    {
        return capacity;
    }

    public void setCapacity(String capacity)
    {
        this.capacity = capacity;
    }

    public int getPrice()
    {
        return price;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    // Metode Print
    public void printDisk()
    {
        System.out.println("==================================");
        System.out.println("Hardisk:");
        System.out.println("==================================");
        System.out.println("Tipe            : " + this.type);
        System.out.println("Kapasitas       : " + this.capacity);
        System.out.println("Harga           : " + "Rp" + this.price);
    }
}
